frontend/src/v5/ui/routes/viewer/drawings/drawingsList/drawingItem/drawingItem.component.tsx

```diff
@@ -78,6 +79,10 @@ export const DrawingItem = ({ drawing, onClick }: DrawingItemProps) => {
 		return () => { deleteAuthUrlFromCache(thumbnailSrc); };
 	}, [latestRevision?._id]);
 
+	useEffect(() => {
+		DrawingRevisionsActionsDispatchers.fetch(teamspace, project, drawingId);
```

Comment:
Why is it necessary to fetch the revisions at this stage?

Reply:
I moved this logic here from `src/v5/ui/routes/viewer/drawings/drawingsList/drawingsListCard.component.tsx` to fix a glitch where: if you are in the viewer, have a drawing selected, so visible in the 2d viewer, and refresh the page, when the drawings card opens next, some drawings do not show the revision data